src/Controller/ArticlesController.php

```diff
@@ -877,12 +877,15 @@ private function articlePageArguments(Request $request, string $id, int $version
                 }
 
                 $exampleRdsArticles = $this->getParameter('example_rds_articles');
-
+                $rdsArticles = $this->getParameter('rds_articles');
+                $rdsInfoBarUrl = '';
                 if (isset($rdsArticles[$item->getId()]) && $this->isGranted('FEATURE_RDS')) {
-                    $infoBars[] = new InfoBar('See this research in an <a href="'.$this->get('router')
-                      ->generate('article-rds', [$item]).'">executable code view</a>.', InfoBar::TYPE_WARNING);
+                    $rdsInfoBarUrl = $this->get('router')->generate('article-rds', [$item]);
                 } elseif (isset($exampleRdsArticles[$item->getId()])) {
-                    $infoBars[] = new InfoBar('This research is available in a <a href="'.$exampleRdsArticles[$item->getId()].'">reproducible view</a>.', InfoBar::TYPE_WARNING);
+                    $rdsInfoBarUrl = $exampleRdsArticles[$item->getId()];
+                }
+                if (strlen($rdsInfoBarUrl) > 0) {
+                  $infoBars[] = new InfoBar('See this research in an <a href="'.$rdsInfoBarUrl.'">executable code view</a>.', InfoBar::TYPE_WARNING);
                 }
```

Comment:
Logic update to display the example RDS article (the Texture-based, current one) only when there is no Stencila-based one available.

Also unifies UI text.